src/models/recipient.js

```diff
@@ -14,6 +14,7 @@ export default (sequelize, DataTypes) => {
       Recipient.hasMany(models.Grant, { as: 'grants', foreignKey: 'recipientId' });
       Recipient.hasMany(models.RttapaPilot, { as: 'rttapaPilots', foreignKey: 'recipientId' });
       Recipient.hasMany(models.SimScoreGoalCache, { foreignKey: 'recipient_id', as: 'recipient' });
+      Recipient.hasMany(models.GoalSimilarityGroup, { foreignKey: 'recipientId', as: 'goalSimilarityGroups' });
```

Comment:
Can we relocate these to the goalSimilarityGroup.js file. In my efforts I have found it prevents possible race conditions to have all associations defined in the file for the model imposing the associations. the only change that will be required is adding "models." before the "Recipient"